licensedb/internal/investigation.go

```diff
@@ -192,17 +192,16 @@ func IsLicenseDirectory(fileName string) bool {
 func ExtractSourceFiles(files []string, fs filer.Filer) [][]byte {
 	candidates := [][]byte{}
 	langs := []string{}
+	var empty []byte
 	for _, file := range files {
-		lang, safe := enry.GetLanguage(file)
-		if safe == true {
+		text, err := fs.ReadFile(file)
+		if err == nil {
+			lang := enry.GetLanguage(file, empty)
```

Comment:
I didn't find relevant information on the enry's README about how to use this function, so I currently use an empty slice. Is this OK?

Reply:
cc @dennwc